yearn/treasury/treasury.py

```diff
@@ -22,6 +23,8 @@
 from yearn.prices.magic import logger as logger_price_magic
 from yearn.utils import contract
 
+from ..constants import STRATEGIST_MULTISIG, TREASURY_WALLETS
```

Comment:
use absolute imports

Reply:
Turns out the objects were already imported in a previous import, I've deleted this relative import